README.md

```diff
@@ -47,25 +67,31 @@ This is useful when you __do not__ want to run all tests in the `%check` section
                % (props.spec_in, strlist_to_shell_cmd(rpm_opts))
```

Comment:
Although not strictly, related to this pull request, we should replace https://github.com/csutils/csmock/blob/main/py/csmock#L283 with a persistent link.

Reply:
I agree, but what do you think about creating a proper `timeout` option for this?

Reply:
As discussed in person, such an option would certainly be useful!

Reply:
https://github.com/aufover/experiments/pull/2/commits/d370d9fdc7571e3f067af0dc02417177da15ac11 is fix for this comment.